xbmc/interfaces/legacy/DrmCryptoSession.h

```diff
@@ -38,44 +38,60 @@ namespace XBMCAddon
     /// Offers classes and functions that allow a developer to work with
     /// DRM-protected contents like Widevine.
     ///
-    /// This type of functionality is closely related to the type of DRM
+    /// This type of functionality is closely related to the type of %DRM
     /// used and the service to be implemented.
     ///
     /// Using the \ref xbmcdrm_CryptoSession "CryptoSession" constructor allow you
-    /// to have access to a DRM session.
-    /// With a DRM session you can read and write the DRM properties
-    /// \ref xbmcdrm_GetPropertyString "GetPropertyString", \ref xbmcdrm_SetPropertyString "SetPropertyString"
+    /// to have access to a %DRM session.
+    /// With a %DRM session you can read and write the %DRM properties
+    /// \ref xbmcdrm_GetPropertyString "GetPropertyString",
+    /// \ref xbmcdrm_SetPropertyString "SetPropertyString"
     /// and establish session keys with
     /// \ref xbmcdrm_GetKeyRequest "GetKeyRequest" and
-    /// \ref xbmcdrm_ProvideKeyResponse "ProvideKeyResponse", or resume previous session keys with 
+    /// \ref xbmcdrm_ProvideKeyResponse "ProvideKeyResponse",
+    /// or resume previous session keys with
     /// \ref xbmcdrm_RestoreKeys "RestoreKeys".
     ///
-    /// When the session keys are established you can use these methods to perform various operations:
-    /// \ref xbmcdrm_Encrypt "Encrypt"/\ref xbmcdrm_Decrypt "Decrypt" for data encryption/decryption,
-    /// \ref xbmcdrm_Sign "Sign"/\ref xbmcdrm_Verify "Verify" for make or verify data-signature.
-    /// Useful for example to implement encrypted communication between a client and the server.
+    /// When the session keys are established you can use these methods
+    /// to perform various operations:
+    /// \ref xbmcdrm_Encrypt "Encrypt" /
+    /// \ref xbmcdrm_Decrypt "Decrypt" for data encryption / decryption,
+    /// \ref xbmcdrm_Sign "Sign" /
+    /// \ref xbmcdrm_Verify "Verify" for make or verify data-signature.
+    /// Useful for example to implement encrypted communication between
+    /// a client and the server.
     ///
-    /// An example where such functionality is useful is the Message Security Layer (MSL)
-    /// transmission protocol used in some VOD applications.
+    /// An example where such functionality is useful is the Message
+    /// Security Layer (MSL) transmission protocol used in some VOD applications.
     /// This protocol (or rather framework) is used to increase the level of security 
-    /// in the exchange of messages (such as licences, manifests or other data) between clients and servers,
-    /// which is a kind of integration to the HTTPS communication standard.
+    /// in the exchange of messages (such as licences, manifests or other data)
```

Comment:
Reading a bit about MSL I found the following: "MSL is used in combination with HTTP versus the previous mix of HTTPS and an in house protocol known as NTBA. This eliminates the latency caused by a SSL/TLS handshake for new connections, allowing MSL to offer similar request/response performance to existing connections."

Hence I think it's probably better to replace "which is a kind of integration to the HTTPS communication standard." with "which defines a security extension/layer on top of the HTTP protocol"

Reply:
done